stable/redis-ha/templates/redis-slave-appliance-vpc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-slave-appliance-vpc
+  annotations:
```

Comment:
Same as above RE: factoring out to values

Reply:
Done.